sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBuilder.java

```diff
@@ -15,6 +17,7 @@
     private String clientSecret;
     private String certificatePath;
     private String certificatePassword;
+    private boolean sharedTokenCacheCred;
```

Comment:
Should have default value “false”

Reply:
its a primitive.
Its default value is false.